templates/cluster-template-clusterclass.yaml

```diff
@@ -108,6 +108,22 @@ spec:
         kind: NutanixClusterTemplate
         matchResources:
           infrastructureCluster: true
+    - jsonPatches:
```

Comment:
There are other envs in the template `KUBEVIP_SVC_ENABLE`, `KUBEVIP_LB_ENABLE` , etc

What do you think about adding a patch with the whole content of this file?
This way we can still have direct variable substation and allow users to override the whole thing if they need to change other parts of it. 

It also has the benefit of keeping the templates clean and reusable in other usecases

Reply:
I agree. Also I don't understand why its being done with sed in a preKubeadmCommnd

Reply:
already tracked thru https://jira.nutanix.com/browse/KRBN-8114 regarding extra parameter related changes